spec/listformat.html

```diff
@@ -202,6 +161,47 @@ <h1>Intl.ListFormat.prototype.resolvedOptions ( )</h1>
         </table>
       </emu-table>
     </emu-clause>
+
+    <emu-clause id="sec-Intl.ListFormat.prototype-toStringTag">
```

Comment:
Noticed in passing:
```suggestion
    <emu-clause id="sec-intl.listformat.prototype-%symbol.tostringtag%" oldids="sec-Intl.ListFormat.prototype-toStringTag">
```

(nd likewise for RelativeTimeFormat (or we could save both for a bigger `id` cleanup that also covers non-symbol sections)